index.html

```diff
@@ -0,0 +1,146 @@
+            coming together.
+          </p>
+          <h3>2023.07.24(MON) ~ 28(FRI)</h3>
+          <p>ENNERCARE CENTRE, TORONTO - CANADA.</p>
+        </div>
+      </section>
+      <section id="programs">
+        <h2>Main Program</h2>
+        <hr class="solid" />
+        <div id="all-programs">
+          <article class="program-cards">
+            <img src="images/programs/program1.png" alt="lecture" />
+            <h2>Lecture</h2>
+            <p>
+              Listen to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program2.png" alt="Exhibition" />
+            <h2>Exhibition</h2>
+            <p>
+              Listens to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program3.png" alt="Forum" />
+            <h2>Forum</h2>
+            <p>
+              Have the time to share your thoughts and opinions with experts
+              for each topic.
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program4.png" alt="Workshop" />
+            <h2>Workshop</h2>
+            <p>
+              Listen to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program5.png" alt="ignite" />
+            <h2>Collision Ignite</h2>
+            <p>
+              Get opportunities to network with Collision affiliates
+              around the World, also after the summit.
+            </p>
+          </article>
+        </div>
+        <button type="button" id="see-button">SEE THE WHOLE PROGRAM</button>
+        <button type="button" id="book-button">BOOK TICKETS</button>
+      </section>
+      <section id="speakers">
+        <h2>Meet our Speakers</h2>
+        <hr class="solid" />
+        <div class="top-speakers"></div>
+        <button type="button" id="more-button">MORE</button>
+      </section>
+      <section id="home-partners">
+        <h2>Partners</h2>
+        <hr class="solid" />
+        <div id="partner-img">
+          <img src="images/partners/logo-accenture.png" alt="accenture" />
+          <img src="images/partners/cisco-logo.png" alt="cisco" />
+          <img src="images/partners/kpmg-logo.png" alt="kpmg" />
+        </div>
+      </section>
+      <footer id="home-footer">
+        <div id="footer-img">
+          <img src="images/Logo/logo.avif" alt="footer logo" />
+        </div>
+        <div id="footer-text">
+          <p>© 2022 Collision Inc. All Rights Reserved</p>
+          <p></p>
+        </div>
+      </footer>
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164409684-13c553df-8fd6-4fed-aec7-a20da54ab6a6.png) | ![image](https://user-images.githubusercontent.com/10332499/164409561-c625f1fd-ac39-4bdd-85d4-01d76c9a1f33.png)

Reply:
resolved